app/carpool/views.py

```diff
@@ -239,6 +239,7 @@ def new():
 @pool_bp.route('/carpools/<uuid>', methods=['GET', 'POST'])
 def details(uuid):
     carpool = Carpool.uuid_or_404(uuid)
+    carpool.has_left = carpool.leave_time < datetime.datetime.now(datetime.timezone.utc)
```

Comment:
Instead of setting it like this, please make this a property on the model object.

Reply:
+1, sorry, been writing too much JavaScript.